### ISSUE TITLE
Add reply_to as part of the JSON request in SparkPostTransport

### ISSUE THREAD

**Issue by byron-smith** [laravel/internals#76]
Hi! Need to include a **reply_to** in the SparkPostTransport as part of the SparkPost API request. 

```
    /**
     * {@inheritdoc}
     */
    public function send(Swift_Mime_Message $message, &$failedRecipients = null)
    {
        ...
        $options = [
            'headers' => [
                'Authorization' => $this->key,
            ],
            'json' => [
                'recipients' => $recipients,
                'content' => [
                    'html' => $message->getBody(),
                    'from' => $this->getFrom($message),
                    'reply_to' => $this->getReplyTo($message),
                    'subject' => $message->getSubject(),
                ],
            ],
        ];
        ...
     }
```

And new getReplyTo method.

```
    /**
     * Get the 'reply_to' headers and format as required by sparkpost
     *
     * @param  Swift_Mime_Message  $message
     * @return string
     */
    protected function getReplyTo(Swift_Mime_Message $message)
    {
        if (is_array($message->getReplyTo())) {
            return current($message->getReplyTo()).' <'.key($message->getReplyTo()).'>';
        }
    }
```

Is this something I can contribute myself or is posting it here recommended?

**Comment by byronsmith77** [comment]
I added a pull request for this regardless. Hopefully I've done everything ok!
https://github.com/laravel/framework/pull/13410

**Comment by tillkruss** [comment]
Can this be closed now, since #13410 got merged?

**Comment by byron-smith** [comment]
Hi, yes, apologies. I will close it. Thanks.
